contracts/multisig-prover/src/encoding/bcs.rs

```diff
@@ -0,0 +1,129 @@
+use bcs::to_bytes;
+use cosmwasm_std::{HexBinary, Uint256};
+use itertools::Itertools;
+use multisig::{key::Signature, msg::Signer};
+
+use crate::{error::ContractError, types::Operator};
+
+#[allow(dead_code)]
+fn encode_proof(
+    quorum: Uint256,
+    signers: Vec<(Signer, Option<Signature>)>,
+) -> Result<HexBinary, ContractError> {
+    let mut operators = make_operators_with_sigs(signers)?;
```

Comment:
sorting is a requirement due to our implementation of the EVM gateway contract. Does the sui contract also require the same?

Reply:
yeah sui requires sorting as well

Reply:
document that on the code?